outputs.tf

```diff
@@ -49,6 +49,7 @@ output "this_rds_cluster_port" {
 output "this_rds_cluster_master_username" {
   description = "The master username"
   value       = element(concat(aws_rds_cluster.this.*.master_username, [""]), 0)
+  sensitive   = true
```

Comment:
https://github.com/hashicorp/terraform-provider-aws/blob/4ea374045df72ac9ac9609f1bee380737967fc8a/aws/resource_aws_rds_cluster.go#L334-L345

`master_password` should be sensitive, but `master_username` should not.

Reply:
Btw, the same logic should have been applied into RDS module I have just merged (https://github.com/terraform-aws-modules/terraform-aws-rds/pull/329 ) where `username` is not sensitive but only `password` is - https://github.com/hashicorp/terraform-provider-aws/blob/main/aws/resource_aws_db_instance.go#L59-L70

Reply:
maybe this is a bug somewhere in the provider. if you switch to terraform 0.15 and then run a plan on one of the examples on `master` right now, you'll see the error. then switch to this PR and run again, its gone - and the error points to `this_rds_cluster_master_username` as you see in #212 - I wonder if it would work if I change `sensitive = false`

Reply:
I trust you on this 100%.

I don't want to spend time on this now but we probably need to pay closer attention to this in the future if it is not possible to identify which attributes are really sensitive just by reading the code (weird).